src/NLog.Extensions.Logging/NLog.Extensions.Logging.csproj

```diff
@@ -72,14 +74,38 @@ For ASP.NET Core, use NLog.Web.AspNetCore: https://www.nuget.org/packages/NLog.W
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="1.0.2" />
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.0.1" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.1.1" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="Microsoft.Extensions.Hosting">
```

Comment:
I wasn't aware this change needed new dependencies. I think it should be a separate package then, what do you think?

Reply:
I want to agree. The usefulness of this functionality is limited to .net core 2.1+. It doesnt really make sense to include include in a multi target project when it literally only builds against .net std 2. However, i know nothing about packages. If you could suggest an assembly name for me though, i'd take a stab at it...